README.md

```diff
@@ -1,6 +1,6 @@
 # hauser
 
-`hauser` is a service to download FullStory data export files and load them into a data warehouse. (Redshift and BigQuery are the only warehouses supported currently. Others are easy to add -- pull requests welcome.)
+`hauser` is a service to download FullStory data export files and load them into a data warehouse. (Redshift and BigQuery are the only warehouses supported currently. Others are easy to add -- pull requests welcome.) There's also an option to save exported files locally.
```

Comment:
maybe update the comment so the new feature of saving the bundles locally is not lost? Just felt that could be missed.

@jameremo Thoughts on something like this? 

`hauser` is a service to download FullStory data export files and load them into a data warehouse. Alternatively data export files can be saved to local disk. Redshift and BigQuery are the only warehouses currently supported. (Others are easy to add -- pull requests welcome.)